### PR TITLE
Requester: Manual req dialog to pick up selected msg

### PR DESCRIPTION
**Pull request by psiinon**
Fixes https://github.com/zaproxy/zaproxy/issues/8365

Also removed the Resed dialog, as thats handled by the Manual Req dialog now.

## Overview
Briefly describe the purpose, goals, and changes or improvements made in this pull request.

## Related Issues
https://github.com/zaproxy/zaproxy/issues/8365

## Checklist
- [x] Update help
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [ ] Write tests
- [ ] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

For more details, please refer to the [developer rules and guidelines](https://www.zaproxy.org/docs/developer/dev-rules-and-guidelines/).

### CODE REVIEW

File: addOns/requester/src/main/java/org/zaproxy/addon/requester/ExtensionRequester.java [hunk 1]
```diff
@@ -150,8 +145,8 @@ private RequesterOptionsPanel getOptionsPanel() {
     }
 
     /*
-     * *Hack to get the selected Message. Used when using the Open in Requester keyboard shortcut.
-     *  It returns the focused message at Requester or the message set in the main Request/Response panel
+     * Hack to get the selected Message. Used when using the Open in Requester keyboard shortcut.
```
Comment (kingthorin) [inline addOns/requester/src/main/java/org/zaproxy/addon/requester/ExtensionRequester.java:148]: 147 should have the double asterisk right?

Comment (kingthorin) [reply addOns/requester/src/main/java/org/zaproxy/addon/requester/ExtensionRequester.java:148]: `/**`

Comment (kingthorin) [reply addOns/requester/src/main/java/org/zaproxy/addon/requester/ExtensionRequester.java:148]: Oh maybe it isn't meant to be javadoc... 🤷‍♂️

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**